tests/testthat/helper-initialize.R

```diff
@@ -3,7 +3,13 @@ library(magrittr)
 library(reticulate)
 library(rlang)
 
-sklearn <- import("sklearn")
+
+
+sklearn <- tryCatch(reticulate::import("sklearn"),
```

Comment:
OK. Although a quick skim through https://github.com/r-lib/devtools/issues/1202 suggests I should just wrap everything, including `library(...)` here, in `if (!interactive() && identical(Sys.getenv("NOT_CRAN"), "true")) { ... }`.

Let me know what you think.

Reply:
I'm not sure that's the best approach. I think using the "ctrl+shift+L" keyboard shortcut in Rstudio is pretty ingrained in most R users package development habits. That defaults to `devtools::load_all()`, so even if `devtools::reload_all()`can do the right thing, there would be some unnecessary friction to make people use it.